plugins/members-area-banking/controllers/banking.coffee

```diff
@@ -57,7 +57,7 @@ class BankingController extends Controller
   _reprocess: (done) ->
     @req.models.Transaction.find().run (err, transactions) =>
       return done err if err
-      regex = /^(.*) M(0[0-9]+) ([A-Z]{3})$/
+      regex = /^(.*) M(?:O|0)*(\d*) ([A-Z]{3})$/
```

Comment:
It needs at least one O/0 (otherwise "M1" would be accepted, which could easily be in a company name); plus it needs at least one digit (otherwise simply "M" would be accepted!); might need to make this a bit more sophisticated... Perhaps

```js
/^(.*) M[0O]+([0-9]+) ([A-Z]{3})$/
```